lua/cinema/misc/cl_floatingnames.lua

```diff
@@ -92,7 +92,7 @@ local function DrawName(ply, opacityScale)
         end
     end
 
-    if ply.NW.bounty > 0 then
+    if ply.NW.bounty and ply.NW.bounty > 0 then
```

Comment:
can do `if (ply.NW.bounty or 0) > 0 then` as the `or` operator will return the second operand if the first is `nil` (unset) or `false`